provider/pkg/provider/http.go

```diff
@@ -154,6 +154,79 @@ func (c *googleHttpClient) sendRequestWithTimeout(method, rawurl string, body ma
 	return result, nil
 }
 
+const multipartBoundary = "boundary-fa78ad331d"
```

Comment:
I assume this value was randomly selected, but it might be good to add a comment to that effect.